WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -555,10 +556,15 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleKeys:(FBRouteRequest *)request
 {
+  XCUIApplication *app = request.session.activeApplication ?: FBApplication.fb_activeApplication;
```

Comment:
To save some processing in case there are multiple active applications, given that the application element itself is not being used for objects operations, maybe we can just use `fb_systemApplication` instead. What do you think?

Reply:
easy